scss/_patterns_contextual-menu.scss

```diff
@@ -16,7 +16,7 @@
     @extend %vf-has-box-shadow;
 
     display: none;
-    margin: 0;
+    margin: 0 0 $input-margin-bottom 0;
```

Comment:
How is margin on the dropdown related to placement?

Contextual menu is absolutely positioned, so I'm not sure if bottom margin on it affects anything?